content/en/docs/reference/config-api/apiserver-admission.v1.md

```diff
@@ -0,0 +1,301 @@
+---
+title: kube-apiserver Admission (v1)
+content_type: tool-reference
+package: admission.k8s.io/v1
+auto_generated: true
```

Comment:
Should this have a `weight`? (might be tracked elsewhere)

Reply:
@sftim I'm aware of the suggestion to add `weight` to these generated files but not sure how that would be possible. Currently, the list of config APIs is manually maintained at `content/en/docs/reference/_index.md`. Maybe we need to add `no_index` to `content/en/docs/reference/config-api/_index.md` or something like that?

Reply:
Let's not let the lack of `weight` block a merge here, then. It could mean unstable sorting between localizations depending on which writing system the localization uses. That's a different issue to the thing we're fixing here.